library/mongo-express

```diff
@@ -1,14 +1,26 @@
-# this file is generated via https://github.com/mongo-express/mongo-express-docker/blob/4b43fe8a1206434cb32a006cd155dd71462f092f/generate-stackbrew-library.sh
+# this file is generated via https://github.com/mongo-express/mongo-express-docker/blob/3971af1b3b2703d9b6f5f6ca12e33ea4ca99bc97/generate-stackbrew-library.sh
 
-Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers)
+Maintainers: Nick Cox <nickcox1008@gmail.com> (@knickers),
+             John Steel <john@jskw.dev> (@BlackthornYugen)
 GitRepo: https://github.com/mongo-express/mongo-express-docker.git
+GitCommit: 3971af1b3b2703d9b6f5f6ca12e33ea4ca99bc97
 
+Tags: 1.0.0-20-alpine3.18, 1.0-20-alpine3.18, 1 latest-20-alpine3.18
+Architectures: amd64, arm64v8
+GitCommit: 5fb537b2fd181e458f58137fdb55d061a8c23da5
+Directory: 1.0/20-alpine3.18
 
-GitCommit: 26e7ea94e4d222de7d52531caee52149ac093c26
-Tags: 1.0.0-alpha.4, 1.0.0-alpha, latest
+Tags: 1.0.0-20, 1.0-20, 1 latest-20, 1.0.0-20-alpine3.17, 1.0-20-alpine3.17, 1 latest-20-alpine3.17
```

Comment:
Ah, I missed all the `1 othertag,` here and on the other entries below.  Not sure where the fix is yet.

Reply:
Just noticed that. We shouldn't ever have a space in the tag right? 

![image](https://github.com/docker-library/official-images/assets/3138467/45980e3b-4076-478f-9da5-60053b78f31d)

Reply:
Nope, no spaces in a tag.  It looks like it comes from the quotes added on https://github.com/mongo-express/mongo-express-docker/pull/100/files#diff-9dd720074107975da3d3140d5e27deb1e3870cb10fd74029ded8d6e1efe98346R85; which references a space separated string, so it was unqoted on purpose.

Reply:
those are two different tags from the script. Still a little bug it seems

Reply:
Good catch @yosifkit

Reply:
My bad! I should have followed my own advice to keep MRs small 😅